pkgs/development/php-packages/parallel/default.nix

```diff
@@ -0,0 +1,22 @@
+{
+  buildPecl,
+  lib,
+  pkgs,
```

Comment:
I forgot this !

Please don't pass `pkgs`, pass `php` instead :)

Reply:
I've an infinite recursion if I pass php instead of pkgs.

Reply:
As discussed on matrix, extension is now marked as broken if ZTS isn't enabled